### ISSUE TITLE
Suddenly no entities rendering

### ISSUE THREAD

**Issue by KeeWolf** [tr7zw/EntityCulling#120]
I've now had this issue multiple times, where all entities stopped rendering. Re-entering the world or even restarting minecraft doesn't fix it. Only when I removed the mod it started working again. The bug seemed to be bound to a world tho.

It possibly has something to do with Spectator Mode, as I always noticed it after I flew around in Spectator, generating new chunks. Not sure tho. Oh and I used the forge mod loader.

Also I used the mod as part of a rather large modpack, so it could certainly be a compatibility issue. [Here](https://wiggly-iguanodon-f6f.notion.site/a3e6bef0efd64879b8a9e3c3de504aeb?v=49f40366ab534a66bdfae58270bada4f&pvs=4) is the entire mod list, in case anyone wants to investigate this further, which I doubt, but I'll give all the information I've got regardless.

I don't have the time or motivation to investigate this further, but I at least wanted to leave a note about it here.

**Comment by tr7zw** [comment]
During spectator mode this mod turns off since walls are kinda just a suggestion during that mode. Potentially you are crashing the culling thread by entering ungenerated chunks or so? But then it would fix itself when restarting the game. It has nothing connecting it to a world or that could survive a restart(unless you mess with the config file), so this report sounds kinda questionable?

**Comment by tr7zw** [comment]
But yea, if you get around to investigateing it more, 4 things:
Check if the values in f3 still update(especially the ms)
See if the debug key does anything(unbound by default)
Verify that the config file is default. 9/10 times that someone messed with it, the mod broke or behaved weirdly. For example trying to increase the culling distance without knowing what you're doing
And lastly provide the log file during the run where its broken.

**Comment by KeeWolf** [comment]
Well... seems like I acted a bit rash, posting this issue here. Turns out, it was something else, as it just happened again without entity culling being installed. It must've been the restart, that fixed it and it seems I just got things confused with it persisting between restarts, but not between worlds.

Sorry, if I wasted your time. Regardless I'm seriously impressed and grateful, how quickly and detailed you replied.
